downstream/modules/aap-hardening/con-logging-log-capture.adoc

```diff
@@ -7,8 +7,8 @@
 
 [role="_abstract"]
 
-Visibility and analytics is an important pillar in Enterprise Security and Zero Trust Architecture. Logging is an important aspect and key to capturing actions and auditing. This can be managed through the built-in audit support by following the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/auditing-the-system_security-hardening[Auditing the system] section of the RHEL Security and Hardening Guide. This is done through the built-in logging support and through the controller built-in Activity Stream support that logs all changes within the controller as well as through the automation logs. More detailed information is available in the link:https://docs.ansible.com/automation-controller/latest/html/administration/logging.html[Logging and Aggregation] section of the {ControllerName} documenation.
+Visibility and analytics is an important pillar of Enterprise Security and Zero Trust Architecture. Logging is key to capturing actions and auditing. This can be managed through the built-in audit support by following the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/auditing-the-system_security-hardening[Auditing the system] section of the Red Hat Enterprise Linux Security and Hardening Guide. Controller's built-in logging and activity stream support logs all changes within controller and automation logs for auditing purposes. More detailed information is available in the link:https://docs.ansible.com/automation-controller/latest/html/administration/logging.html[Logging and Aggregation] section of the {ControllerName} documenation.
```

Comment:
Edit link title "Red Hat Enterprise Linux Security and Hardening Guide" to "Security hardening for Red Hat Enterprise Linux"